src/types.d.ts

```diff
@@ -63,9 +63,9 @@ export interface EnterpriseCustomerUser {
 export interface EnterpriseLearnerData {
   enterpriseCustomer: Types.EnterpriseCustomer | null;
   activeEnterpriseCustomer: Types.EnterpriseCustomer | null;
-  allLinkedEnterpriseCustomerUsers: EnterpriseCustomerUser[];
+  allLinkedEnterpriseCustomerUsers: EnterpriseCustomerUser[] | [];
   staffEnterpriseCustomer: Types.EnterpriseCustomer | null;
-  enterpriseFeatures: Types.EnterpriseFeatures;
+  enterpriseFeatures: Types.EnterpriseFeatures | {};
```

Comment:
Similarly here, the `EnterpriseFeatures` type is defined as an interface, which assumes an object. The fallback `| {}` is not needed here.